### PR TITLE
Post-PR review. 

### PR DESCRIPTION
**Pull request by tdykstra**
Review of the article created by #32529 

Mostly changes to conform to the style guide. 
On periods in bulleted lists, see https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=56575

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/fundamentals/minimal-apis/aspnetcore-openapi.md](https://github.com/dotnet/AspNetCore.Docs/blob/11675822e527352508d3b1e5f0a355d644179355/aspnetcore/fundamentals/minimal-apis/aspnetcore-openapi.md) | [Get started with Microsoft.AspNetCore.OpenApi](https://review.learn.microsoft.com/en-us/aspnet/core/fundamentals/minimal-apis/aspnetcore-openapi?branch=pr-en-us-32562) |

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
Search and replace here -- too verbose
```## Add and configure OpenAPI document generation

ADD the OpenAPI-related services to `Program.cs`: <!-- REPLACE FOLLOWING WITH SNIPPET -->```

**Comment by Rick-Anderson** [comment]
image

**Comment by Rick-Anderson** [comment]
> . The requests below will resolve the v1 and internal documents respectively.
The following requests resolve (nuke will)

**Comment by Rick-Anderson** [comment]
> Transformers are useful for scenarios like:
* adding parameters to all operations in a document, 
* modifying descriptions for parameters or operations, 
* adding top-level information to the OpenAPI document.

**Review by Rick-Anderson — CHANGES_REQUESTED** [review]
rewrite

Operation transformers apply to each individual operation (combination of path + HTTP method) and can be used to modify parameters or responses on endpoints.

Operation transformers apply to each individual operation, modifying parameters or responses on endpoints. These operations are a combination of the path and the HTTP method.

LInk to source on new APIs, ie
https://github.com/dotnet/aspnetcore/blob/main/src/OpenApi/src/Transformers/IOpenApiDocumentTransformer.cs#L9

**Comment by Rick-Anderson** [comment]
https://github.com/dotnet/aspnetcore/blob/main/src/OpenApi/src/Services/OpenApiOptions.cs

File: aspnetcore/fundamentals/minimal-apis/aspnetcore-openapi.md [hunk 1]
```diff
@@ -128,7 +128,12 @@ Transformers fall into two categories:
 * Document transformers have access to the entire OpenAPI document and can be used to make global modifications to the document.
 * Operation transformers apply to each individual operation (combination of path + HTTP method) and can be used to modify parameters or responses on endpoints.
 
-Transformers can be registered onto the document via the `UseTransformers` call on the `OpenApiOptions` object.
+Transformers can be registered onto the document via the `UseTransformers` call on the `OpenApiOptions` object. The following snippet shows different ways to register transformers onto the document:
```
Comment (Rick-Anderson) [inline aspnetcore/fundamentals/minimal-apis/aspnetcore-openapi.md:131]:
```suggestion
Transformers can be registered onto the document via the `UseTransformer` call on the `OpenApiOptions` object. The following snippet shows different ways to register transformers onto the document:
```

**Comment by Rick-Anderson** [comment]
useTransformer in https://github.com/dotnet/aspnetcore/blob/main/src/OpenApi/src/Services/OpenApiOptions.cs#L47

**Review by Rick-Anderson — COMMENTED** [review]
aspnetcore\src\Mvc\Mvc.ApiExplorer\src\ApiDescriptionGroupCollectionProvider.cs
contains ApiDescriptionGroups 

Probably create another PR to update these ref when the API is built. It's nice to have a link to the source and the API ref.

File: aspnetcore/fundamentals/minimal-apis/aspnetcore-openapi.md [hunk 2]
```diff
@@ -159,36 +157,35 @@ builder.Services.AddOpenApi(options =>
...
+Service-activated document transformers can also be used to implement transformers that rely on instances from DI to modify the application. The sample below demonstrates a document transformer that uses the `IAuthenticationSchemeProvider` service from the authentication layer to check if any JWT bearer-related schemes are registered in the application and add them to the OpenAPI document's top level.
 
 [!code-csharp[](~/fundamentals/minimal-apis/9.0-samples/WebMinOpenApi/Program.cs?name=snippet_documenttransformer2)]
 
-Document transformers are unique to the document instance they are associated with. In the example below, a transformer registers authentication-related requirements to the `internal` document but leaves the `public` document unmodified.
+Document transformers are unique to the document instance they are associated with. In the example below, a transformer registers authentication-related requirements to the `internal` document but leaves the `public` document unmodified.<!-- "example below" this is the end of the section, there is no following snippet. need to find snippet or delete reference to it -->
```
Comment (captainsafia) [inline aspnetcore/fundamentals/minimal-apis/aspnetcore-openapi.md:176]:
```suggestion
Document transformers are unique to the document instance they are associated with. In the example below, a transformer registers authentication-related requirements to the `internal` document but leaves the `public` document unmodified.

[!code-csharp[](~/fundamentals/minimal-apis/9.0-samples/WebMinOpenApi/Program.cs?name=snippet_multidoc_operationtransformer1)]
```

**Pull request merged**